src/GameSelector/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { Button, Center, FormControl, Input, VStack } from "native-base";
+import React from "react";
+
+export interface GameSelectorFormData {
+  gameId: number | undefined;
+}
+
+const GameSelectorForm = () => {
+  const [formData, setData] = React.useState<GameSelectorFormData>({
+    gameId: undefined,
+  });
+  const [errors, setErrors] = React.useState({});
+
+  const validate = () => {
+    if (formData.gameId === undefined || formData.gameId === 0) {
+      setErrors({
+        ...errors,
+        gameId: "The game id cannot be empty",
+      });
+      return false;
+    }
+
+    setErrors({});
+    return true;
+  };
+
+  const onSubmit = () => {
+    validate() ? console.info("Submitted") : console.error("Validation Failed");
+  };
+
+  return (
+    <VStack width="90%" mx={3}>
+      <FormControl isRequired isInvalid={'gameId' in errors}>
+        <FormControl.Label _text={{ bold: true }}>Game id</FormControl.Label>
+        <Input
+          placeholder="1, 2..."
+          keyboardType="numeric"
+          returnKeyType="done"
+          onChangeText={(value) => setData({ ...formData, gameId: +value })}
+        />
+
+        {"gameId" in errors ? (
```

Comment:
Is this `in` specific to react native ?

Reply:
No, this is the in operator: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in

Reply:
Cool. Well I learned something today, it's an operator I never use :)